daemon/src/projection.rs

```diff
@@ -828,7 +974,7 @@ impl State {
     }
 
     async fn update_cfd(&mut self, db: db::Connection, id: OrderId) -> Result<()> {
-        let cfd = db.load_cfd(id, self.network).await?;
+        let cfd = db.load_open_cfd(id, self.network).await?;
```

Comment:
⚠️ we have to take `closed` cfds into account here

I think we can expose `load_cfd` and use that here... then we always get the CFD no matter if closed or open.
Otherwise we will need some condition here as well, to be able to load the cfd. 
This is somewhat critical, because otherwise once we moved a cfd to closed we will error when loading, because the order_id won't be found anymore.

Reply:
It's good that you noticed this behaviour, but I think I disagree with your conclusion.

We call `update_cfd` when we receive a new `CfdEvent` for a CFD. A closed CFD does not need to be updated any further. We don't expect any more events and, more importantly, we don't want to record any more events.

Reply:
> 

hm... interesting point. So we would just keep the last state in the projection and that CFD will only be loaded again on restart. Sounds legit to me. Let's get it in then and see what happens 😄